examples/pipelines/fl_cross_silo_factory/fl_factory.py

```diff
@@ -0,0 +1,503 @@
+"""Factory to build a basic FL pipeline."""
```

Comment:
Would be good to have a very clear instruction (can be a separate readme about using such fl_factory to write/submit fl_pipeline experiment) about: whether users need to touch fl_factory.py, and if yes, when and how? Ideally, modify the config file would be enough for users, and touch fl_factory.py API code is not needed (except for some very rare case).

Reply:
For now I'd suggest do not touch factory at all, but I will add a disclaimer in the header to explain why.